release_images_inference.yml

```diff
@@ -63,3 +63,29 @@ release_images:
       example: False
       disable_sm_tag: False
       force_release: False
+  6:
+    framework: "pytorch"
```

Comment:
Is it tensorflow release or Pytorch that is being focused on this PR? PR Heading says TF, but I see PT being modified.

Reply:
The schema for TF has been added.